js/PulsarFormComponent.js

```diff
@@ -61,6 +63,13 @@ PulsarFormComponent.prototype.init = function () {
     // choice block click behaviour
     choiceBlock.on('change', '.controls input[type="checkbox"], .controls input[type="radio"]', component.selectionButtons);
 
+    // initialise tinepickers
+    var $timePickers = this.$html.find('[data-timepicker=true]');
+
+    $.each($timePickers, function() {
```

Comment:
Could we check whether `timepicker` handles a jQuery collection (I suspect it might based on the usual jQuery plugin convention, although I'm making an assumption here)? 

If the case is that it _does_ handle a jQuery collection, this loop here is not necessary and we can go ahead and pass `$timePickers` through as a collection to `timePickerComponent.bindTimePicker`.